### DISCUSSION TITLE
Google Fonts locally

### DISCUSSION THREAD

**Discussion by krychaxp** [vercel/next.js#24738] [Ideas]
### Describe the feature you'd like to request

When user want use google fonts, nextjs should download all this fonts locally and add style+link tags to next/head

### Describe the solution you'd like

use (and/or improve) something like [nextjs-google-fonts](https://www.npmjs.com/package/nextjs-google-fonts), and make it as a default setting in nextjs

To see how its work enter to my website [krychaxp.pl](https://krychaxp.pl)

### Describe alternatives you've considered

build your own idea

**Comment by sachinraja** [comment]
Correct me if I'm wrong, but isn't the google CDN going to be faster than having your app serve the fonts (on a production deployment)? You can just import it:
```css
@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap")
```
If you need this for offline use, you can download the font or use your package. I don't think that should be handled by next.js.
